spec.md

```diff
@@ -558,6 +563,18 @@ message VolumeMetadata {
   // each Plugin keeps this information as small as possible.
   map<string, string> values = 1;
 }
+
+// A standard way to encode credential data. The total bytes of the values in
+// the Data field must be less than MaxSecretSize bytes.
+message Credentials {
+  // Data contains the credential data, for example username and password. Each
+  // key must consist of alphanumeric characters, '-', '_' or '.'. The
+  // serialized form of the secret data is a base64 encoded string, representing
```

Comment:
Ah right. I am going to change `bytes` to `string`. That way string data can be passed through without worrying about character encoding, and binary data can be passed in encoded base64 or quoted-printable.